### PR TITLE
samples: bluetooth: nrf_dm: Added support for Thingy53

### PR DESCRIPTION
**Pull request by hlord2000**
Adds overlay and configuration for Thingy53.

### CODE REVIEW

**Comment by greg-fer** [comment]
Adding @peknis for related doc update monitoring. Can you coordinate with @KAGA164 and @grochu ?

**Comment by hlord2000** [comment]
Updating after 2.4.0 release. I have signed the CLA but it hasn't updated here.

File: samples/bluetooth/nrf_dm/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.overlay [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			role = "remote";
+			zephyr,buffer-size = <2048>;
+			status = "okay";
+		};
+	};
+
+
+	/* The selected pins will be used by Distance Measurement module for debug purposes.
+	 *
+	 * Note: Pay attention to select the same GPIO pins
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.overlay:52]:
```suggestion
	 * Note: Make sure to select the same GPIO pins
```

File: samples/bluetooth/nrf_dm/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.overlay [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			mboxes = <&mbox 2>, <&mbox 3>;
+			mbox-names = "rx", "tx";
+			role = "remote";
+			zephyr,buffer-size = <2048>;
+			status = "okay";
+		};
+	};
+
+
+	/* The selected pins will be used by Distance Measurement module for debug purposes.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/child_image/hci_rpmsg/boards/thingy53_nrf5340_cpunet.overlay:50]:
```suggestion
	/* The selected pins will be used by the Distance Measurement module for debug purposes.
```

File: samples/bluetooth/nrf_dm/boards/thingy53_nrf5340_cpuapp.overlay [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			zephyr,buffer-size = <2048>;
+			status = "okay";
+		};
+	};
+};
+
+ /* Enable pin forwarding to network core. The selected pins will be used by
+  * Distance Measurement module for debug purposes.
+  *
+  * Note: Pay attention to assign the same GPIO pins as those provide in network core DTS overlay.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/boards/thingy53_nrf5340_cpuapp.overlay:53]:
```suggestion
  * Note: Make sure to assign the same GPIO pins as those provide in network core DTS overlay.
```

File: samples/bluetooth/nrf_dm/boards/thingy53_nrf5340_cpuapp.overlay [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+			mbox-names = "tx", "rx";
+			role = "host";
+			zephyr,buffer-size = <2048>;
+			status = "okay";
+		};
+	};
+};
+
+ /* Enable pin forwarding to network core. The selected pins will be used by
+  * Distance Measurement module for debug purposes.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/boards/thingy53_nrf5340_cpuapp.overlay:51]:
```suggestion
  * the Distance Measurement module for debug purposes.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**